hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/CertificateClient.java

```diff
@@ -121,14 +130,23 @@ boolean verifySignature(byte[] data, byte[] signature,
   X509Certificate queryCertificate(String query);
 
   /**
-   * Stores the Certificate.
+   * Stores the Certificate  for this client. Don't use this api to add
+   * trusted certificates of other s.
    *
-   * @param certificate - X509 Certificate
-
+   * @param pemEncodedCert        - pem encoded X509 Certificate
+   * @param force                 - override any existing file
+   * @param isLocalIdentityCert   - true if certificate belongs to the
+   * identity cert for this certificate client.
    * @throws CertificateException - on Error.
+   *
+   * Note: Certificate client can store certificates for other daemons as well.
+   * Local certificate refers to the certificate issued to this certificate
+   * client. This is stored along with public key and private key.
+   * Certificate of other daemons is stoed in sub dirs named after
+   * certificate serial id of certificate.
```

Comment:
should we store non-local certificate under a common root?

Reply:
Comment is not correct, patch stores all certificate in default certs dir for simplicity. Updated the comment.